scripts/docker/Dockerfile.openmptarget

```diff
@@ -14,7 +14,7 @@ RUN apt-get update && apt-get install -y \
 
 ARG NPROC=8
 
-ARG CMAKE_VERSION=3.10.3
+ARG CMAKE_VERSION=3.18.5
```

Comment:
This is needed for building `clang`? I was seeing a similar requirement for `SYCL+CUDA`.

Reply:
LLVM requires 3.13+
I grabbed the latest of the 3.18 series because I see it as a reasonable choice.